docs/aqua-book/language/crdt-maps.md

```diff
@@ -0,0 +1,81 @@
+# CRDT Maps
+
+A map is a kind of [collection](types.md#collection-types) that store arrays of elements that can be accessible by `string` keys. Under the hood maps are appendable collections (streams) of key-pairs.
```

Comment:
What is the point of saying that a map stores arrays if it does not?
`get` basically generates the array on the fly, it can be written even in Aqua itself:
```
func get(m: %u32, k: string) -> []u32:
  res: *u32
  for km, v <- m:
    if km == k:
      res <<- v
   <- res

```